apps/web/src/custom-pages/weekly-newsletter-page.tsx

```diff
@@ -28,12 +28,27 @@ function NewsItemContent({ content }: { content?: EditorState }) {
   );
 }
 
-function NewsSection({ newsItem }: { newsItem: NewsItem }) {
+async function NewsSection({ newsItem }: { newsItem: NewsItem }) {
+  const t = await getScopedI18n("weeklyNewsletter");
   const content = newsItem.content as unknown as EditorState;
 
   return (
     <article>
       <h3 id={stringToId(newsItem.title)}>{newsItem.title}</h3>
+      {newsItem.linkToSignUp ? (
```

Comment:
We could move this inside the `class="prose"` `<NewsItemContent />`, so that it gets the right vertical margins automatically. Or add the margins here so it doesn't look out of place